README.rst

```diff
@@ -106,13 +106,47 @@ Quick Install
 
 Install the package with pip::
 
-    $ pip install pyroomacoustics
+    pip install pyroomacoustics
 
-The requirements are::
+A [cookiecutter](https://github.com/fakufaku/cookiecutter-pyroomacoustics-sim)
+is available that generates a working simulation script for a few 2D/3D
+scenarios::
 
-* numpy 
-* scipy 
-* matplotlib
+    # if necessary install cookiecutter
+    pip install cookiecutter
+
+    # create the simulation script
+    cookiecutter gh:fakufaku/cookiecutter-pyroomacoustics-sim
+
```

Comment:
Do you want to add that to then run the script you can simply do `python <chosen_script_name>.py`?

Also I don't know if this can be prevented but when I run the cookiecutter to create the simulation script, the command line interface doesn't raise an warning/error when I provide a `script_name` if there's already a file with that name. I only get an error after inserting all the other parameters and the cookiecutter tries to create the script:

```
A file with the same name already exists
ERROR: Stopping generation because post_gen_project hook script didn't exit successfully
Hook script failed (exit status: 1)
```

If possible, it could be good to add a condition when entering the `script_name` to see if the file already exists OR when creating the script it uses a new file name, e.g. `<chosen_script_name_001>`, and lets the user know.

Reply:
> Do you want to add that to then run the script you can simply do python <chosen_script_name>.py?
I've thought about that. Why not!

I haven't really dug into the internals of cookiecutter, but testing the filename before creation doesn't seem to be out of the box. However, there is an option to save the input in case you run the command again.